amqp/subscriber.go

```diff
@@ -142,3 +144,48 @@ func (s *DefaultSubscriber) subscribe(key string) (<-chan AMQP.Delivery, error)
 	}
 	return s.consume(queue)
 }
+
+// ConsumeMessages consumes all messages in a specific queue and try to handle them as uplinks or events depending on the
+// routing key.
+func (s *DefaultSubscriber) ConsumeMessages(queue string, uplinkHandler UplinkHandler, eventHandler DeviceEventHandler) error {
+	messages, err := s.consume(s.name)
```

Comment:
Didn't payed attention when I copy pasted the previous code from `ConsumeUplink`.